src/guides/v2.3/release-notes/release-notes-2-3-5-open-source.md

```diff
@@ -492,8 +504,6 @@ We have fixed hundreds of issues in the Magento 2.3.5 core code.
 
 *  JavaScript bundling and JavaScript minimization should be disabled by default when `Magento_Baler` is enabled.
```

Comment:
Hmm, this is confusing, since the [baler module](https://github.com/magento/baler) is not part yet of Magento by default. It is also still in an alpha state, so this feels strange to find this here in the release notes?

Reply:
I'll follow up on this one, too. Thanks.